.gitignore

```diff
@@ -35,7 +35,4 @@ Makefile
 *.swp
 
 # ARM Compute Library
-dep/ComputeLibrary/build/examples
-dep/ComputeLibrary/build/opencl-1.2-stubs
-dep/ComputeLibrary/build/src
-dep/ComputeLibrary/build/tests
+dep/ComputeLibrary/build
```

Comment:
I think that a script to download Arm Compute Library binaries (shared objects) is required.